src/utils/businessHours.test.ts

```diff
@@ -487,4 +488,72 @@ describe('businessHours tests', function () {
       );
     });
   });
+
+  describe('getBusinessHoursLeft', function () {
+    it('should correctly calculate business hours left overnight', function () {
+      const triageBy = '2023-12-22T18:00:00.000Z';
+      const now = moment('2023-12-22T01:00:00.000Z');
+      expect(
+        getBusinessHoursLeft(
```

Comment:
When a function signature is more than ~3 args, I think named parameters are the way to go. In particular, the last 3 arguments here are all the same type (strings), and as someone unfamiliar with the codebase, it's quite hard to figure out what corresponds to what just by reading the invocation. 

What if we did:
```
// At definition-site:
interface BusinessHoursLeft {
  triageBy: string,
  now: moment.Moment,
  repo: string,
  org: string,
  productArea: string
}
function getBusinessHoursLeft({ triageBy, now, repo, org, productArea }: BusinessHoursLeft {
  ...
}

// At call-site:
getBusinessHoursLeft({
          triageBy,
          now,
          repo: 'test-ttt-simple',
          org: 'getsentry',
          productArea: 'Other'
        })
```

Now I know what's what, and can catch errors that the type system can't (ex, swapped string args) more easily when scanning.

Reply:
The reason why I did not do this is because it seemed explicit enough what was going on where the function is being called:

```
const businessHoursLeft = getBusinessHoursLeft(
        triageBy,
        now,
        repo,
        org,
        productArea
      );
```

This ambiguity would only exist for testing. This is not a super widely used function, so I'd rather not declare an interface for it. I'll clear up what the strings are by giving them names before passing them in if that's helpful.

Reply:
Sure, but that relies on the call-site naming its variables properly. If you use an interface, this is required, rather than optional, enforcing more readable code. The language even encourages this practice if you use an interface (you can just type the var name without stuttering), so it feels like a best practice.

In any case, I trust your judgment, just a small nit as a rather unfamiliar reader of the code.

Reply:
There are other functions in the file that could use cleaning up too. Creating an issue to get those resolved in a later PR:
https://github.com/getsentry/eng-pipes/issues/737